src/assets/css/style.css

```diff
@@ -0,0 +1,341 @@
+@import url('https://fonts.googleapis.com/css2?family=El+Messiri&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Material+Icons&display=swap");
+
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
```

Comment:
Hi! Great work on your project, however, I think your css code could be more DRY. I would suggest that instead of writing all the things you need to style in this manner, you can simply create a class in your html, add the style to your class then apply the class to all these elements from where they are located in your code. That way your code would be shorter and cleaner